README.md

```diff
@@ -63,4 +63,4 @@ development VM it's possible to increase the `proxy_read_timeout` value in
 
 Tests can run in browser at `/specs`
 
-Or in terminal to run only the jasmine tests you can use `RAILS_ENV=test bundle exec rake jasmine:ci`
+Or in terminal to run only the jasmine tests you can use `RAILS_ENV=test bundle exec rake jasmine:ci` or in Docker `govuk-docker run -e RAILS_ENV=test static-lite bundle exec rake jasmine:ci`
```

Comment:
If the image (?) being run is `static-lite` could this also be run from the directory itself using...

```
govuk-docker-run -e RAILS_ENV=test bundle exec rake jasmine:ci
```

This doesn't really matter, its more to do with how govuk-docker works. Feel free to ignore if you don't think it's worth it.

Reply:
Hmm, I get an error when I run the shortcut version - I think the the shortcut runs:

```sh
govuk-docker run -e static-lite RAILS_ENV=test bundle exec rake jasmine:ci
```

which throws because `-e` should be before `static-lite` - so `-e` gets interpreted as an executable file:

```sh
Error response from daemon: OCI runtime create failed: container_linux.go:370: starting container process caused: exec: "-e": executable file not found in $PATH: unknown
```

I'd like to leave this as is rather than spend time figuring out a way of making the shortcut work correctly.